base/cvd/cuttlefish/common/libs/utils/BUILD.bazel

```diff
@@ -138,3 +140,23 @@ cc_test(
         "@googletest//:gtest_main",
     ],
 )
+
+cc_library(
+    name = "files_test_helper",
+    testonly = 1,
+    srcs = [
+        "files_test_helper.cpp",
+    ],
+    hdrs = [
+        "files_test_helper.h",
```

Comment:
Can `files_test_helper.cpp` and `files_test_helper.h` be part of the `utils_test` `srcs` member?

Reply:
Tried that first but couldn't make it work. Bazel complains that files_test_helper.h can't be found (as included in files_test.cpp).

Reply:
Probably because the `cc_test` is missing the `strip_include_prefix` attribute.